batch10/assignment3/HASSAN_SHAIKH.md

```diff
@@ -0,0 +1,26 @@
+Q1) Complete all the labs - Done
+
+Q2) How to leverage cache using Dockerfiles 
+>> Dockerfile help to avoid downloading image again and again.
```

Comment:
Its not image, its layers as each command represents layers so the commands that are less likely to be changed should be placed on top of Dockerfile